src/java.base/share/classes/java/text/RuleBasedCollator.java

```diff
@@ -591,7 +591,7 @@ public synchronized CollationKey getCollationKey(String source)
         // Here's a hypothetical example, with the collation element represented as
         // a three-digit number, one digit for primary, one for secondary, etc.
         //
-        // String:              A     a     B   \u00e9 <--(e-acute)
+        // String:              A     a     B   é
```

Comment:
Maybe "é (\u00e9, e-acute)"?

Reply:
I used the `U+` code point notation here as well. I think that is better to make sure there are no misunderstandings, and no risk of the escaped character inadvertently being converted into an actual unicode character down the line by processors.